src/color/creating_reading.js

```diff
@@ -108,9 +108,22 @@ p5.prototype.blue = function(c) {
  * rect(50, 20, 35, 60);
  * </code>
  * </div>
+ * <div>
+ * <code>
+ * noStroke();
+ * colorMode(HSB, 255);
+ * let c = color('hsb(60, 100%, 50%)');
+ * fill(c);
+ * rect(15, 20, 35, 60);
+ * let value = brightness(c); // A 'value' of %50 is 127.5
```

Comment:
For English the percentage symbol should go after the number (50% instead of %50)

Reply:
Yes, sorry I missed that.  
Should I change that and put in a new pull request,  I'm new to working with git repos.

Reply:
@hellonearthis you can make the change directly and push your commit to your own repo, and it will update here